packages/devtools_app/lib/main.dart

```diff
@@ -2,32 +2,14 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+
 import 'package:flutter/material.dart';
-import 'package:flutter/foundation.dart';
-import 'package:html_shim/html.dart' show document, window;
 
+import 'src/config_specific/flutter/framework_initialize/framework_initialize.dart';
 import 'src/flutter/app.dart';
-import 'src/framework/framework_core.dart';
 
 void main() {
```

Comment:
the one non-mechanical change is moving this code to framework_initialize.dart and its conditional imports.